### ISSUE TITLE
crest not reading in the correct number of threads

### ISSUE THREAD

**Issue by O2-AC** [polyluxus/runxtb.bash#45]
When using `runxtb -p48 -C crest` the script will correctly set the queuing system parameters and export the correct environment variables for xtb. However, `crest` will not pick those environment variables up and only run with 4 processors. Not sure where that setting is coming from, though.

Example when using crest directly (exactly as in the generated submission script):
```
$ crest coord
...
 Estimated runtime for one MTD (9.0 ps) on a single thread: 7 min 46 sec
 Estimated runtime for a batch of 14 MTDs on 4 threads: 31 min 4 sec
...
```
Example when using crest explicitly with the `-T` option:
```
$ crest coord -T 24
...
 Estimated runtime for one MTD (9.0 ps) on a single thread: 17 min 30 sec
 Estimated runtime for a batch of 14 MTDs on 24 threads: 17 min 30 sec
...
```

**Comment by polyluxus** [comment]
I'll need to check how xtb is doing this and how it is different in crest.

I'm currently not sure how to support that, but I guess we'll find out.

As a workaround you can pass the option directly. It's redundant and possibly has some drawbacks, but it should work.

```
runxtb debug -p 6 -s -C crest -- coord -T 6
```

Should work. But I'll investigate.

**Comment by polyluxus** [comment]
It appears to me, that crest doesn't utilise environment variables, but xtb does.

I've changed the "hotfix" from #46. It now includes `-T threads` automagically into the command line that's passed on to crest when it matches the executable. 
I also moved this to a place where it takes affect for all queues and interactive, too. (Hopefully.)

I wouldn't call this a permanent solution, but it should work for the meantime. (It should be robust enough, too.)
